schemas/asset_schema.json

```
@@ -181,6 +181,15 @@
         "minItems": 2,
         "maxItems": 2
       }
+    },
+    "customProperties" : {
+      "type" : "array",
+      "description" : "Names of custom properties used somewhere in this asset.",
+      "items" : {
+        "type" : "string",
+      },
+      "uniqueItems" : true,
+      "default" : []
```

```suggestion
    "customProperties": {
      "type": "object",
      "description" : "Non-standardized asset properties for custom tools or tool chains.",
      "propertyValues": {
        "type": "string"
      }
    }
```

I would propose to do it like this, as this allows adding any number of key-value pairs as properties, like in this example:

```JSON
"customProperties": {
  "test": "my first custom property",
  "test2": "my second custom property"
}
````